src/tox/config/__init__.py

```diff
@@ -807,6 +807,11 @@ def passenv(testenv_config, value):
             passenv.add("PROCESSOR_ARCHITECTURE")  # platform.machine()
             passenv.add("USERPROFILE")  # needed for `os.path.expanduser()`
             passenv.add("MSYSTEM")  # fixes #429
+
```

Comment:
I think this empty line is not necessary :-)

Reply:
Hmm. Perhaps you're right... The thing I'm trying to communicate with the newline is that the comment applies to a group of items. What I really want is for the next variable added to put a blank line before it so that the group of three items is clear from the syntax. I'll just remove it and let future people figure it out however.

Reply:
Fixed in aa94f9d9.